internal/v1/api.yaml

```diff
@@ -1111,3 +1129,28 @@ components:
           type: string
           format: uuid
           example: '123e4567-e89b-12d3-a456-426655440000'
+    DistributionProfileResponse:
+      type: array
+      description: |
+        List of profiles for a given distribution
+      items:
+        $ref: '#/components/schemas/DistributionProfileItem'
+    DistributionProfileItem:
+      type: string
+      enum: [ xccdf_org.ssgproject.content_profile_anssi_bp28_enhanced,
+        xccdf_org.ssgproject.content_profile_anssi_bp28_high,
+        xccdf_org.ssgproject.content_profile_anssi_bp28_intermediary,
+        xccdf_org.ssgproject.content_profile_anssi_bp28_minimal,
+        xccdf_org.ssgproject.content_profile_cis,
+        xccdf_org.ssgproject.content_profile_cis_server_l1,
+        xccdf_org.ssgproject.content_profile_cis_workstation_l1,
+        xccdf_org.ssgproject.content_profile_cis_workstation_l2,
+        xccdf_org.ssgproject.content_profile_cui,
+        xccdf_org.ssgproject.content_profile_e8,
+        xccdf_org.ssgproject.content_profile_hipaa,
+        xccdf_org.ssgproject.content_profile_ism_o,
+        xccdf_org.ssgproject.content_profile_ospp,
+        xccdf_org.ssgproject.content_profile_pci-dss,
+        xccdf_org.ssgproject.content_profile_standard,
```

Comment:
Standard is only supported by `fedora` so maybe we can remove it.
For reference: https://www.osbuild.org/guides/image-builder-on-premises/oscap-remediation.html

Reply:
Ah, maybe we can keep it there ? I mean it's only in the possible profiles, true we can't use it yet, but maybe it's worth it being exhaustive ?